hub/powertoys/grouppolicy.md

```diff
@@ -374,6 +374,271 @@ If you disable this policy, the user won't be able to enable Enable paste with A
+- Example value: `<disabled/>`
+
+#### Original user interface is available
+
+Supported on PowerToys 0.83.0 or later.
+
+This policy configures if the user can use the old Mouse Without Borders user interface.
+
+If you enable or don't configure this policy, the user takes control over the setting and can enable or disable the old user interface.
+
+If you disable this policy, the user won't be able to enable the old user interface.
+
+##### Group Policy (ADMX) information
+
+- GP unique name: MwbUseOriginalUserInterface
+- GP name: Original user interface is available
+- GP path: Administrative Templates/Microsoft PowerToys/MouseWithoutBorders
+- GP scope: Computer and user
+- ADMX file name: _PowerToys.admx_
+
+##### Registry information
+
+- Path: Software\Policies\PowerToys
+- Name: MwbUseOriginalUserInterface
+- Type: DWORD
+- Example value: `0x00000000`
+
+##### Intune information
+
+- OMA-URI: `./Device/Vendor/MSFT/Policy/Config/PowerToys~Policy~PowerToys~MouseWithoutBorders/MwbUseOriginalUserInterface`
+- Example value: `<disabled/>`
+
+#### Validate remote machine IP Address
+
+Supported on PowerToys 0.83.0 or later.
+
+This policy configures if reverse DNS lookup is used to validate the remote machine IP Address.
+
+If you enable this policy, the setting is enabled and the IP Address is validated.
+
+If you disable this policy, the setting is disabled and the IP Address is not validated.
+
+If you don't configure this policy, the user takes control over the setting and can enable or disable it.
+
+##### Group Policy (ADMX) information
+
+- GP unique name: MwbValidateRemoteIp
+- GP name: Validate remote machine IP Address
+- GP path: Administrative Templates/Microsoft PowerToys/MouseWithoutBorders
+- GP scope: Computer and user
+- ADMX file name: _PowerToys.admx_
+
+##### Registry information
+
+- Path: Software\Policies\PowerToys
+- Name: MwbValidateRemoteIp
+- Type: DWORD
+- Example value: `0x00000001`
+
+##### Intune information
+
+- OMA-URI: `./Device/Vendor/MSFT/Policy/Config/PowerToys~Policy~PowerToys~MouseWithoutBorders/MwbValidateRemoteIp`
+- Example value: `<enabled/>`
+
+#### Predefined IP Address mapping rules
+
+Supported on PowerToys 0.83.0 or later.
+
+This policy allows you to define IP Address mapping rules.
+
+If you enable this policy, you can define IP Address mapping rules that the user can't change or disable.
+Please enter one mapping per line in the format: "hostname IP"
+
+If you disable or don't configure this policy, no predefined rules are applied.
+
+##### Group Policy (ADMX) information
+
+- GP unique name: MwbPolicyDefinedIpMappingRules
+- GP name: Predefined IP Address mapping rules
+- GP path: Administrative Templates/Microsoft PowerToys/MouseWithoutBorders
+- GP scope: Computer and user
+- ADMX file name: _PowerToys.admx_
+
+##### Registry information
+
+- Path: Software\Policies\PowerToys
+- Name: MwbPolicyDefinedIpMappingRules
+- Type: DWORD
```

Comment:
@alvinashcraft
I made a mistake in this line: It has to be `MULTI_SZ` instead of `DWORD`. Can you please open a fix PR. That would be great. Thank you.

Reply:
@htcfreek No problem. I'll do that now.